ravenpy/models/emulators/blended.py

```diff
@@ -0,0 +1,835 @@
+                        :SoilEvaporation SOILEVAP_TOPMODEL       SOIL[0]        ATMOSPHERE      # AET
+          :EndProcessGroup CALCULATE_WTS {params.par_r05}
+          #                              para_r05
+          :Convolve                      CONVOL_GAMMA            CONVOLUTION[0] SURFACE_WATER   # 'surface runoff'
+          :Convolve                      CONVOL_GAMMA_2          DELAYED_RUNOFF SURFACE_WATER   # 'delayed runoff'
+          :ProcessGroup #quickflow group
+                        :Baseflow        BASE_LINEAR_ANALYTIC    SOIL[1]        SURFACE_WATER
+                        :Baseflow        BASE_POWER_LAW          SOIL[1]        SURFACE_WATER
+          :EndProcessGroup CALCULATE_WTS {params.par_r06}
+          #                              para_r06
+          :ProcessGroup #snow balance group
+                        :SnowBalance     SNOBAL_HMETS            MULTIPLE       MULTIPLE
+                        :SnowBalance     SNOBAL_SIMPLE_MELT      SNOW           PONDED_WATER
+                        :SnowBalance     SNOBAL_HBV              MULTIPLE       MULTIPLE
+                        #:SnowBalance    SNOBAL_GAWSER           MULTIPLE       MULTIPLE
+          :EndProcessGroup CALCULATE_WTS {params.par_r07} {params.par_r08}
+          #                              para_r07 para_r08
+        :EndHydrologicProcesses
+
+        #:CreateRVPTemplate
+
+        #---------------------------------------------------------
+        # Output Options
+        #
+        #:WriteForcingFunctions
+        :EvaluationMetrics {evaluation_metrics}
+        :WriteNetcdfFormat  yes
+        #:NoisyMode
+        :SilentMode
+        :PavicsMode
+        {suppress_output}
+
+        :NetCDFAttribute title Simulated river discharge
+        :NetCDFAttribute history Created on {now} by Raven
+        :NetCDFAttribute references  Craig, J.R., and the Raven Development Team, Raven user's and developer's manual (Version 2.8), URL: http://raven.uwaterloo.ca/ (2018).
+        :NetCDFAttribute comment Raven Hydrological Framework version {raven_version}
+
+        :NetCDFAttribute model_id hmets
+
+        :NetCDFAttribute time_frequency day
+        :NetCDFAttribute time_coverage_start {start_date}
+        :NetCDFAttribute time_coverage_end {end_date}
+        """
+        self.config.rvi.set_tmpl(rvi_tmpl)
+
+        self.config.rvi.params = self.config.rvp.params
+
+        #########
+        # R V C #
+        #########
+
+        rvc_tmpl = """
+        # initialize to 1/2 full
+        # x(29)*1000/2
+        :UniformInitialConditions SOIL[0] {TOPSOIL_hlf}
+        # x(30)*1000/2
+        :UniformInitialConditions SOIL[1] {PHREATIC_hlf}
+
+        :HRUStateVariableTable (formerly :InitialConditionsTable)
+           :Attributes SOIL[0] SOIL[1]
+           :Units mm mm
+           1 {TOPSOIL_hlf} {PHREATIC_hlf}
+           # x(29)*1000/2       x(30)*1000/2
+        :EndHRUStateVariableTable
+        """
```

Comment:
I think this mechanism conflicts or duplicate the hru_state and basin_state initialization mechanism.

Reply:
I think I fixed it.